R/robustbayesianmetaanalysis.R

```diff
@@ -416,119 +413,119 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
-  jasp_table$addColumnInfo(name = "lowerCI", title = gettext("Lower"), type = "number", overtitle = CI_overtitle)
-  jasp_table$addColumnInfo(name = "upperCI", title = gettext("Upper"), type = "number", overtitle = CI_overtitle)
+  jaspTable$addColumnInfo(name = "terms",   title = "",               type = "string")
+  jaspTable$addColumnInfo(name = "mean",    title = gettext("Mean"),  type = "number")
+  jaspTable$addColumnInfo(name = "median",  title = gettext("Median"),type = "number")
+  jaspTable$addColumnInfo(name = "lowerCI", title = gettext("Lower"), type = "number", overtitle = overtitleCI)
+  jaspTable$addColumnInfo(name = "upperCI", title = gettext("Upper"), type = "number", overtitle = overtitleCI)
   
   if (individual) {
-    jasp_table$addColumnInfo(name = "error",   title = gettext("MCMC error"), type = "number")
-    jasp_table$addColumnInfo(name = "ess",     title = gettext("ESS"),        type = "integer")
-    jasp_table$addColumnInfo(name = "rhat",    title = gettext("Rhat"),       type = "number")
+    jaspTable$addColumnInfo(name = "error",   title = gettext("MCMC error"), type = "number")
+    jaspTable$addColumnInfo(name = "ess",     title = gettext("ESS"),        type = "integer")
+    jaspTable$addColumnInfo(name = "rhat",    title = gettext("Rhat"),       type = "number")
   }
   
   
-  if (is.null(results_table))
-    return(jasp_table)
+  if (is.null(resultsTable))
+    return(jaspTable)
   
   # fill rows
-  temp_i <- 0
-  for (i in c(1:nrow(results_table))[grepl("theta", rownames(results_table))]) {
-    temp_i <- temp_i + 1
-    temp_row <- list(
-      terms    = add_info[["study_names"]][temp_i],
-      mean     = results_table[i, "Mean"],
-      median   = results_table[i, "Median"],
-      lowerCI  = results_table[i, if(individual) ".025" else as.character(.5 - options[["results_CI"]] / 2)],
-      upperCI  = results_table[i, if(individual) ".975" else as.character(.5 + options[["results_CI"]] / 2)]
+  tempI <- 0
+  for (i in c(1:nrow(resultsTable))[grepl("theta", rownames(resultsTable))]) {
+    tempI <- tempI + 1
+    tempRow <- list(
+      terms    = add_info[["study_names"]][tempI],
+      mean     = resultsTable[i, "Mean"],
+      median   = resultsTable[i, "Median"],
+      lowerCI  = resultsTable[i, if(individual) ".025" else as.character(.5 - options[["resultsCI"]] / 2)],
+      upperCI  = resultsTable[i, if(individual) ".975" else as.character(.5 + options[["resultsCI"]] / 2)]
     )
     if (individual) {
-      temp_row[["error"]] <- results_table[i, "MCMC error"]
-      temp_row[["ess"]]   <- results_table[i, "ESS"]
-      temp_row[["rhat"]]  <- results_table[i, "Rhat"]
+      tempRow[["error"]] <- resultsTable[i, "MCMC error"]
+      tempRow[["ess"]]   <- resultsTable[i, "ESS"]
+      tempRow[["rhat"]]  <- resultsTable[i, "Rhat"]
     }
     
-    jasp_table$addRows(temp_row)
+    jaspTable$addRows(tempRow)
   }
   
   
   # add footnote
   if (add_info[["effect_size"]] %in% c("r", "d", "OR")){
```

Comment:
Wouldn't you want to change this to "effectSize"?

Reply:
no, you can't because it's taken from the fitted RoBMA object, so it needs to use naming specified there